keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -594,6 +597,19 @@ would allow us to delete the pod and trying again with a new one, but is not don
 at the moment because admission checks cannot be retried if a check finds
 a transient problem.
 
+**Note:** The immediate motivation for standardizing attributes largely stems
+from the current behavior of the `MatchAttribute` constraint, which relies on
+exact value matching. While this KEP provides a solution for many cross-driver
+alignment needs, a more flexible long-term solution is envisioned with
+[KEP-5254: DRA support for MatchExpression
+constraints](https://github.com/kubernetes/enhancements/issues/5254) (a
+work-in-progress). That proposal aims to introduce a `MatchExpression`
+constraint, allowing devices to be evaluated against CEL expressions, which will
+enable more complex and dynamic selection criteria. However, solving this
+critical alignment problem today is essential for latency-sensitive workloads.
+Standardizing attributes helps achieve this without introducing any conflicts
+with the future capabilities of KEP-5254.
+
```

Comment:
In any case, even with #5254 (which is now called "Constraints with CEL"), we will need common attributes to do cross-driver comparisons.